RPI/co2-monitor.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
systemd file to start the monitor script when RPI booted. (ref: https://www.thedigitalpictureframe.com/ultimate-guide-systemd-autostart-scripts-raspberry-pi/)

To configure the service (run these commands in RPI):

```
# 1. Copy the file to /etc/systemd/system/
sudo cp co2-monitor.service /etc/systemd/system/co2-monitor.service

# 2. Start the service
sudo systemctl start co2-monitor.service

# 3. Enable service to start on boot
sudo systemctl enable co2-monitor.service
```

Reply:
cc @obisi